ReleaseNotes.md

```diff
@@ -0,0 +1,12 @@
+## Third Party Software ##
+
+This is a list of publicly available software that is redistributed, categorized by license:
+
+### BSD License ###
+* [Fluid Infusion v1.5 (pre-release)](http://fluidproject.org)
```

Comment:
If we are using the latest from master it's probably more accurately described as Fluid Infusion v2.0 (pre-release)